jinad/api/endpoints/pea.py

```diff
@@ -80,25 +79,6 @@ def streamer(generator):
         logger.info("Exiting from Pod log_iterator")
 
 
-@router.get(
-    path='/log',
```

Comment:
we can delete **GET /log** from `pod` & **Websocket /wslogs** from `flow`

Reply:
yes that is the idea

Reply:
are they currently required? will it break something?

Reply:
with removal of `log_iterator`, **GET /log** is already broken

Reply:
yes but I mean, will this PR break something already working? will it require some changes in core? or only the endpoint of logs?